### DISCUSSION TITLE
能否加入给用户分配多个目录，每个目录分配不同的权限

### DISCUSSION THREAD

**Discussion by fastxteam** [AlistGo/alist#2693] [Ideas]
一般上Alist会挂载多个网盘，对于admin来说拥有所有的访问权限

问题：
目前用户设置无需密码访问后，可以访问设定目录下的所有信息
也可以缩小用户访问目录的范围

但是有以下需求，可能会增加用户体验度：

比如以下结构：
/
阿里云盘
   - 用户A （密码访问）
   - 共享内容
   - 用户C （密码访问）
百度云盘
  - 用户D （密码访问）
  - 用户E
  - 共享内容

创建了用户A，我想让A在目录（用户A）中拥有全部的权限，同时又可以访问阿里云盘，百度云盘共享内容

1. 假如说我限制了用户A的根路径为/阿里云盘/用户A/ 那么其就不能访问其他的内容
2. 假如我设置了用户A根路径为/，那么给其无密码访问就造成了所有都能查看
3. 如果设置访客权限都能看，A独有目录（用户A）所有权限，那每次完成需求需要切换账号

所以能否对用户增加多个目录访问的权限，然后设定主目录，对每个目录拥有的权限都能个性化定制之类的。

**Comment by AbcEric** [comment]
把共享内容放在/，在未登录状态下，可以查看共享内容。登录或输入密码后，查看自己的私有空间。

**Reply by fastxteam** [reply]
这么做在自己的目录和共享目录下需要频繁切换账号

**Reply by zerofo** [reply]
怎么定义 自己的私有空间？
